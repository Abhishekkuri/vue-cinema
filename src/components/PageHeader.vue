<template>
  <div class="header-container">
    <img src="@/assets/cinema-trans.png" alt="logo" />
  </div>
</template>

<script>
export default {};
</script>

<style lang="scss" scoped>
.header-container {
  display: flex;
  align-items: center;
  justify-content: center;
  background-color: $color-primary;
  img {
    height: 20rem;
    width: 20rem;
  }
}
</style>
