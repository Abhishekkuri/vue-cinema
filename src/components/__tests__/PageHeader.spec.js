import { shallowMount } from "@vue/test-utils";
import PageHeader from "@/components/PageHeader";

describe("PageHeader.vue", () => {
  test("Header renders correctly", () => {
    expect(shallowMount(PageHeader).element).toMatchSnapshot();
  });
  test("Header renders logo", () => {
    expect(shallowMount(PageHeader).contains("img")).toBe(true);
  });
});
