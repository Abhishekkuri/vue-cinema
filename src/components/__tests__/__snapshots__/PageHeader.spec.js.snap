// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader.vue Header renders correctly 1`] = `
<div
  class="header-container"
>
  <img
    alt="logo"
    src="@/assets/cinema-trans.png"
  />
</div>
`;
